en-US/Players_Guide/03_Creating_a_Character/Classes/Cleric/Sacred_Mandate.md

```diff
@@ -0,0 +1,161 @@
+  \
+  As a bonus action, you cloak a creature that you choose within 30 feet in illusions.
+  Until the start of your next turn, each target has partial cover and can take the Hide action as a bonus action.
+  \
+  **6th Level:**
+  The number of affected creatures increases by your proficiency bonus.
+- **Channel Divinity: Aegis of Faith** (Life, War)
+  \
+  When you gain this feature, choose a damage type.
+  As an action, you conjure a divine shield around an ally within 30 feet.
+  Until the end of their next turn, the target has resistance to the chosen damage type.
+  \
+  **6th Level:**
+  The target also gains temporary hit points equal to `2d6 + your Wisdom modifier` when you use this feature.
+- **Channel Divinity: Echo of Command** (Trickery, War)
+  \
+  As a bonus action, choose a creature whose location you know within 30 feet.
+  - If the creature is an ally, it can immediately use its reaction to make one weapon attack or cast a cantrip.
+    It adds your Wisdom modifier to the attack roll.
+  - If the creature is an enemy, it must make a Wisdom saving throw.
+    On a failure, it uses its reaction to make a weapon attack or a spell attack with a cantrip, targeting a creature you choose.
+
+  **6th Level:**
+  When you use this ability, you may choose whether affected creatures can be frightened or charmed until the end of their next turn.
+
+###### Divine Strike or Potent Spellcasting (8th & 14th Level)
+
+Choose one of the following:
+
+- **Divine Strike.**
+  When you gain this feature, choose a damage type.
+  Your weapon attacks deal an extra `1d8` damage of the chosen type, increasing to `2d8` at 14th level.
+  \
+  **Suggested Damage Types:**
+  - **Knowledge:** Psychic
+  - **Life:** Radiant
+  - **Light:** Radiant
+  - **Storm:** Elemental or Thunder
+  - **Trickery:** Toxic
+  - **War:** Force or Thunder
+- **Potent Spellcasting.**
+  You add your Wisdom modifier to the damage you deal with cleric cantrips.
+
+###### Exalted Mandate (17th Level)
+
+Choose one matching your mandate:
+
+- **Avatar of War** (War, Storm)
+  \
+  When reduced to 0 hit points, you drop to 1 hit point instead and may use your reaction to make one weapon attack.
+  You also gain resistance to bludgeoning, piercing, and slashing damage for 1 minute.
+- **Illuminated Soul** (Light, Life)
+  \
+  As an action, you emit a 30 foot aura for 1 minute.
+  Allies in the area gain advantage on saving throws against being blinded, charmed, or frightened and regain 1d6 hit points at the start of their turns.
+- **Shifting Facade** (Trickery, Knowledge)
+  \
+  The locations of anyone within 30 feet of you become difficult for enemies to determine for one minute, as all creatures that you choose, including you, seem to switch places with each other.
+  Enemies must succeed on a Wisdom saving throw or target a random creature when trying to attack any creature within range.
+
+You have one use of this feature, regaining it after a long rest.
```

Comment:
Since I'm using the parentheses after the feature name for the level you gain a feature, I move the "once per long rest" to the bottom and adapted the wording used in other "X times per long rest" features (in this case: Bardic Inspiration).